charmtools/build/__init__.py

```diff
@@ -673,6 +674,10 @@ def main(args=None):
             build.check_series()
 
         build()
+
+        lint, exit_code = proof.proof(os.getcwd(), False, False)
+        if lint:
+            print("\n".join(lint))
```

Comment:
I think it would be nicer to use `log.warn` to match the colorization of the rest of the output.